hooks/scripts/run-codespell.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 staged_files=$(git diff --staged --name-only HEAD --diff-filter=ACMR)
-riot -v run -s hook-codespell $staged_files
+if [ -z "$staged_files" ]; then
+    riot -v run -s hook-codespell $staged_files
+else
+    echo 'Run codespell skipped: No Python files were found in git diff --staged'
```

Comment:
```suggestion
```
Don't think this should run if there's no files in staging

Reply:
I made one more change. Codespell is now run on all files so I changed `echo 'Run codespell skipped: No Python files were found in git diff --staged'` to `echo 'Run codespell skipped: No files were found in git diff --staged'`